test/ui-test/testSuites/suite_status/shared/steps/chatSteps.py

```diff
@@ -68,6 +68,11 @@ def step(context, message):
 def step(context, message_index, message):
     _statusChat.reply_to_message_at_index(message_index, message)
     _statusChat.verify_last_message_sent(message)
+    
+@Then("the user can edit the message at index |any| and add to the end \"|any|\"" )
```

Comment:
WDYT if we modify it like this to avoid the `and` since, using `and`,  makes me think it should be in another statement?

```suggestion
@Then("the user can edit the message at index |any| adding \"|any|\" to the end" )
```